syntax = "proto3";

enum MSGID
{
    MSG_BEGIN = 0;
    MSG_TEST_ID = 1005 ;
    MSG_PLAYER_MOVE_ID = 1006 ;
    MSG_PLAYER_ATTACK_ID = 1007 ;
    MSG_PLAYER_REGISTE_ID = 1008 ;
    MSG_PLAYER_LOGIN = 9999;

    MSG_ITEM_ADD = 10000;
    MSG_ITEM_REMOVE = 10001;
    MSG_PLAYER_FIND = 10002;
    MSG_RANKLIST_SHOW = 10003;

    MSG_LOGIN_AUTH_PASSED = 10004;
    MSG_LOGIN_AUTH_FAILED = 10005;
    MSG_LOGIN_REQUEST_DB = 10006;
    MSG_LOGIN_REPLIY_CLIENT = 10007;
};
